Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/cmd/edit.go

```diff
@@ -294,14 +295,15 @@ outter:
 			}
 
 			if hold, msg := jsonmerge.TestPreconditionsHold(patch, preconditions); !hold {
-				fmt.Fprintf(out, "error: %s", msg)
+				fmt.Fprintf(out, "Error: %s\n", msg)
```

Comment:
We assume that errors printed by fmt.Errorf or errors.New will be preceded by something else hence the convention to lowercase those. This is starting a new line, and I am also consistent with https://github.com/kubernetes/kubernetes/blob/8b186991e2a85731943e099efcd3c676a6f1c774/pkg/kubectl/cmd/edit.go#L400-L406 which are also going to be printed to out.